.teamcity/src/main/kotlin/util/UpdateWrapper.kt

```diff
@@ -0,0 +1,80 @@
+package util
+
+import common.BuildToolBuildJvm
+import common.Os
+import common.VersionedSettingsBranch
+import common.javaHome
+import common.requiresOs
+import common.toCapitalized
+import jetbrains.buildServer.configs.kotlin.AbsoluteId
+import jetbrains.buildServer.configs.kotlin.BuildType
+import jetbrains.buildServer.configs.kotlin.ParameterDisplay
+import jetbrains.buildServer.configs.kotlin.buildSteps.exec
+import jetbrains.buildServer.configs.kotlin.triggers.finishBuildTrigger
+import promotion.FINAL_RELEASE_BUILD_CONFIGURATION_ID
+import promotion.RELEASE_CANDIDATE_BUILD_CONFIGURATION_ID
+import vcsroots.useAbsoluteVcs
+
+object UpdateWrapper : BuildType({
+    name = "UpdateWrapper"
+    id("UpdateWrapper")
+
+    val vcsBranch = VersionedSettingsBranch.fromDslContext()
+
+    vcs.useAbsoluteVcs(vcsBranch.vcsRootId())
+
+    requirements {
+        requiresOs(Os.LINUX)
+    }
+
+    params {
+        text(
+            "wrapperVersion",
+            "should-be-overridden",
+            display = ParameterDisplay.PROMPT,
+            allowEmpty = false,
+            description =
+                "The version of Gradle to update to. " +
+                    "Can be a specific version or one of 'latest', 'release-candidate', 'release-milestone', 'release-nightly', 'nightly'.",
```

Comment:
💅 `'release-nightly', 'nightly'` are not useful as the result can't be merged to any release{7x,8x,} / master branch